app/styles/ui/_no-branches.scss

```diff
@@ -1,6 +1,7 @@
 .no-branches {
+  width: 365px;
+  margin: var(--spacing) auto;
   display: flex;
-  flex: 1;
```

Comment:
Both for the Branches and Pull Requests empty states, I decided to keep the fixed width of 365px instead of letting the imagery, text and button expand to fill the container. Specifically, in the case of Branches tab, the Create New Branch could get uncomfortably wide.  See comparison below.


#### Full width empty state
![2024-01-28-002298@2x](https://github.com/desktop/desktop/assets/3560/f974965a-a6df-4006-a505-b90ced01d09f)

#### Fixed width empty state
![2024-01-28-002299@2x](https://github.com/desktop/desktop/assets/3560/29ad7c55-0912-4e36-b1ed-db22174829da)